src/Components/WebAssembly/Build/test/BuildIntegrationTests/BuildLazyLoadTest.cs

```diff
@@ -40,7 +40,7 @@ public async Task Build_LazyLoadExplicitAssembly_Debug_Works()
             var bootJson = ReadBootJsonData(result, Path.Combine(buildOutputDirectory, "wwwroot", "_framework", "blazor.boot.json"));
 
             // And that it has been labelled as a dynamic assembly in the boot.json
-            var dynamicAssemblies = bootJson.resources.dynamicAssembly;
+            var dynamicAssemblies = bootJson.resources.lazyAssembly;
```

Comment:
I thought I moved these tests?

Reply:
We discussed this offline.

Turns out there's two instances of this test file in our repo. I submitted #23777 to remove this one and will rebase this PR with that change.